Source/Lib/Codec/EbPictureBufferDesc.c

```diff
@@ -142,35 +139,19 @@ EB_ERRORTYPE EbReconPictureBufferDescCtor(
 	pictureBufferDescPtr->lumaSize		= (pictureBufferDescInitDataPtr->maxWidth + pictureBufferDescInitDataPtr->leftPadding + pictureBufferDescInitDataPtr->rightPadding) *
 									      (pictureBufferDescInitDataPtr->maxHeight + pictureBufferDescInitDataPtr->topPadding + pictureBufferDescInitDataPtr->botPadding);
     pictureBufferDescPtr->chromaSize    = pictureBufferDescPtr->lumaSize >> (3 - pictureBufferDescInitDataPtr->colorFormat);
-    pictureBufferDescPtr->packedFlag    = EB_FALSE;
-
-    pictureBufferDescPtr->strideBitIncY  = 0;
-    pictureBufferDescPtr->strideBitIncCb = 0;
-    pictureBufferDescPtr->strideBitIncCr = 0;
 
+    pictureBufferDescPtr->bufferEnableMask = pictureBufferDescInitDataPtr->bufferEnableMask;
     // Allocate the Picture Buffers (luma & chroma)
     if(pictureBufferDescInitDataPtr->bufferEnableMask & PICTURE_BUFFER_DESC_Y_FLAG) {
-        EB_CALLOC(EB_BYTE, pictureBufferDescPtr->bufferY, pictureBufferDescPtr->lumaSize      * bytesPerPixel * sizeof(EB_U8) + (pictureBufferDescPtr->width + 1) * 2 * bytesPerPixel*sizeof(EB_U8), 1, EB_N_PTR);
-        pictureBufferDescPtr->bufferY += (pictureBufferDescPtr->width+1) * bytesPerPixel;
-    }
-    else {
-        pictureBufferDescPtr->bufferY   = 0;
+        EB_CALLOC_ALIGNED_ARRAY(pictureBufferDescPtr->bufferY, pictureBufferDescPtr->lumaSize * bytesPerPixel);
```

Comment:
@hassount, the old size is
pictureBufferDescPtr->lumaSize * bytesPerPixel + (pictureBufferDescPtr->width + 1) * 2 * bytesPerPixel, and it is not aligned mem. 
New one keeps sync with AV1, which I thinks makes more sense.
Which should I use for recon buffer?